pom.xml

```diff
@@ -75,17 +75,17 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-cps</artifactId>
-        <version>2.29</version>
+        <version>2.33-20170525.192309-1</version> <!-- TODO: update to release once 2.33 is released -->
       </dependency>
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-job</artifactId>
-        <version>2.9</version>
+        <version>2.11</version>
```

Comment:
Nope, I deliberately switched from the `2.12-TIMESTAMPYSTUFF` upstream PR of `workflow-job` (that will hopefully still get released at some point, since I do still think there's a valid use case for `FlowExecutionListener`, if not here) to `2.11`. =)